languages/csharp/exercises/concept/numbers/.docs/after.md

```diff
@@ -7,14 +7,17 @@ int i = 123;
 double d = 54.29;
 ```
 
-Arithmetic is done using the standard [arithmetic operators][arithmetic-operators] (`+`, `-`, `*`, etc.). Numbers can be compared using the standard [comparison operators][comparison-operators] (`<`, `>=`, etc.).
+Arithmetic is done using the standard [arithmetic operators][arithmetic-operators] (`+`, `-`, `*`, etc.). Numbers can be compared using the standard [comparison operators][comparison-operators] (`<`, `>=`, etc.) and the [equality-][equality-operators] (`==`) and [inequality][equality-operators] (`!=`) operators.
```

Comment:
Intentional dash in `[equality-]` but not `[inequality]` ?

Reply:
It was intentional, as it is referring to the operators suffix of the "inequality" word. So instead of

"and the equality operator (`==`) and the inequality (`!=`) operator"

I do:

"and the equality- (`==`) and the inequality (`!=`) operators"

Is this perhaps not correct?

Reply:
Strictly speaking I am not sure for these particular words. Normally I would usually would hyphenate on the first one if the second also used it. e.g. "left- and right-handed".
@workingjubilee might know for sure.

Reply:
I've updated the text